doc/source/whatsnew/v0.24.0.txt

```diff
@@ -168,6 +168,120 @@ Current Behavior:
     ...
     OverflowError: Trying to coerce negative values to unsigned integers
 
+read_html Incompatibilities
+---------------------------
+
+:func:`read_html` previously ignored ``colspan`` and ``rowspan`` attributes.
+Now it understands them, treating them as a sequence of cells with the same
+value.
+
+Previous Behavior:
+
+.. code-block:: ipython
+
+    In [1]: pd.read_html("""
```

Comment:
to avoid repeating the code, make an ipython block above where you define the html itself). then the preivious uses a code-block to show the output, and the new uses an ipython block

Reply:
Ah, I get it now. (I actually read the docs to preview what it looks like.) Thanks -- looks like a super improvement.